projects/cloud/app/services/change_user_role_service.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class ChangeUserRoleService < ApplicationService
+  attr_reader :user_id, :organization_id, :role, :acting_user
+
+  module Error
+    Unauthorized = Class.new(StandardError)
+  end
+
+  def initialize(user_id:, organization_id:, role:, acting_user:)
+    super()
+    @user_id = user_id
+    @organization_id = organization_id
+    @role = role
+    @acting_user = acting_user
+  end
+
+  def call
+    user = User.find(user_id)
```

Comment:
This will [raise an error](https://api.rubyonrails.org/v6.1.4/classes/ActiveRecord/RecordNotFound.html) if the user cannot be found. I'd rescue from it, and return a custom error that we can then map to an error response in the GraphQL layer. I'd then add a unit test for it.